pkg/common/executions.go

```diff
@@ -1,29 +1,20 @@
 package common
 
 import (
-	"math/rand"
+	"fmt"
 
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
-)
-
-const ExecutionIDLength = 10
 
-// In kubernetes, resource names must comply with this regex: '[a-z]([-a-z0-9]*[a-z0-9])?'
-const AllowedExecutionIDStartCharStr = "abcdefghijklmnopqrstuvwxyz"
-const AllowedExecutionIDStr = "abcdefghijklmnopqrstuvwxyz1234567890"
+	"k8s.io/apimachinery/pkg/util/rand"
+)
 
-var AllowedExecutionIDStartChars = []rune(AllowedExecutionIDStartCharStr)
-var AllowedExecutionIDChars = []rune(AllowedExecutionIDStr)
+const ExecutionIDLength = 20
+const ExecutionStringFormat = "f%s"
```

Comment:
thoughts on having the start char be something different than 'f' so we can easily distinguish between propeller-launched executions vs. admin-generated names?

Reply:
Make sense. Will make change in flytectl too so that we can distinguish those aswell.

Reply:
awesome, thank you!